spotify_dl/utils.py

```diff
@@ -1,10 +1,10 @@
-def sanitize(name, replace_with=''):
+def sanitize(name, replace_with=""):
     """
     Removes some of the reserved characters from the name so it can be saved
     :param name: Name to be cleaned up
     :return string containing the cleaned name
     """
-    clean_up_list = ["\\", "/", ":", "*", "?", "\"", "<", ">", "|", "\0", "$"]
+    clean_up_list = ["\\", "/", ":", "*", "?", '"', "<", ">", "|", "\0", "$"]
```

Comment:
why was the backspace removed? Backspaces are reserved characters in Windows

Reply:
sorry let me add them back